examples/platforms/nrf52840/misc.c

```diff
@@ -49,6 +49,13 @@ __WEAK void nrf5CryptoDeinit(void)
     // This function is defined as weak so it could be overridden with external implementation.
 }
 
+void otPlatCommissioningClkChange(otInstance * aInstance, otClockSpeed aSpeed)
+{
+  (void)aInstance;
+  (void)aSpeed;
+  // TODO: Write me!
```

Comment:
nit: I think some platform will not implement this functionality. I would prefer to remove this TODO, or make a comment that this function is internationally empty.